server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/MappingRoutes.java

```diff
@@ -97,5 +101,21 @@ public void define(Service service) {
                 .haltError();
         }
     }
+
+    @GET
+    @Path(USER_MAPPING_PATH + "{user}")
+    @ApiOperation(value = "Getting all user mappings in RecipientRewriteTable")
+    @ApiResponses(value = {
+        @ApiResponse(code = HttpStatus.OK_200, message = "OK", response = List.class),
+        @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = "Invalid user parameter values.")
+    })
+     private List<MappingValueDTO> getUserMappings(Request request, Response response) throws RecipientRewriteTableException {
+        User user = User.fromUsername(request.params("user").toLowerCase());
```

Comment:
`"user"` should be a class level constant that we reuse also for route definition and swagger documentation